src/RoleObjects/__tests__/ButtonData.test.js

```diff
@@ -77,6 +77,48 @@ describe('ButtonData', () => {
       });
     });
 
+    describe('"addChild" and "addChildAt"', () => {
+      let childObj;
+      const errorMsg = /Children of button cannot be interactive/;
+      beforeEach(() => {
+        childObj = new createjs.Shape();
+      });
+      describe('child is not an Accessible object', () => {
+        it('"addChild" throws error if child is not an Accessible object', () => {
+          expect(() => {
+            cjsButton.accessible.addChild(childObj);
+          }).toThrowError(errorMsg);
+        });
+
+        it('"addChildAt" throws error if child is not an Accessible object', () => {
+          expect(() => {
+            cjsButton.accessible.addChildAt(childObj, 0);
+          }).toThrowError(errorMsg);
+        });
+      });
+
+      describe('child has the SPAN role', () => {
+        beforeEach(() => {
+          AccessibilityModule.register({
+            displayObject: childObj,
+            role: AccessibilityModule.ROLES.SPAN,
+          });
+        });
+
+        it('"addChild" does not throw an error if child has the SPAN role', () => {
+          expect(() => {
+            cjsButton.accessible.addChild(childObj);
+          }).not.toThrowError(errorMsg);
+        });
+
+        it('"addChildAt" does not throw an error if child has the SPAN role', () => {
+          expect(() => {
+            cjsButton.accessible.addChildAt(childObj, 0);
+          }).not.toThrowError(errorMsg);
+        });
+      });
+    });
```

Comment:
- added child checking to bring our coverage back up over the threshold that we lost when we excluded `__jestSharedSetup` (rather than reducing the min coverage %)